webpack/config.js

```diff
@@ -1,13 +1,17 @@
 var path = require('path');
 
-module.exports = {
+module.exports = (publicPath = "/") => ({
   entry: {
-    app: path.resolve(__dirname, '../src/main.js')
+		app: path.resolve(__dirname, '../src/main.js'),
+		// this creates a duplicate output bundle for local serving as the app expects output bundle to be in the 'react-foundation-docs/' sub-directory
```

Comment:
Consistent spaces instead of tabs:
```suggestion
    app: path.resolve(__dirname, '../src/main.js'),
    // this creates a duplicate output bundle for local serving as the app expects output bundle to be in the 'react-foundation-docs/' sub-directory
```